### PR TITLE
Upgrade to Karaf 4.4.5

### PR DESCRIPTION
**Pull request by holgerfriedrich**
For release notes, see:
https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12353604&projectId=12311140

Refs: openhab/openhab-core#4133

### CODE REVIEW

File: launch/app/app.bndrun [hunk 1]
```diff
@@ -123,8 +123,6 @@ feature.openhab-model-runtime-all: \
 # done
 #
 -runbundles: \
-	org.ops4j.pax.logging.pax-logging-api;version='[2.2.3,2.2.4)',\
-	org.ops4j.pax.logging.pax-logging-log4j2;version='[2.2.3,2.2.4)',\
```
Comment (wborn) [inline launch/app/app.bndrun:127]: IIRC the Pax Logging bundles are at the top so those bundles are started first so the log levels are applied immediately which prevents noisy debug/trace logging at startup.

File: launch/app/app.bndrun [hunk 2]
```diff
@@ -296,4 +248,52 @@ feature.openhab-model-runtime-all: \
 	org.openhab.core.voice;version='[4.2.0,4.2.1)',\
 	org.openhab.ui;version='[4.2.0,4.2.1)',\
 	org.openhab.ui.basic;version='[4.2.0,4.2.1)',\
-	org.openhab.ui.iconset.classic;version='[4.2.0,4.2.1)'
+	org.openhab.ui.iconset.classic;version='[4.2.0,4.2.1)',\
+	com.google.guava;version='[33.0.0,33.0.1)',\
+	com.google.guava.failureaccess;version='[1.0.2,1.0.3)',\
+	com.sun.jna;version='[5.14.0,5.14.1)',\
```
Comment (wborn) [inline launch/app/app.bndrun:254]: I always keep the bundles sorted as sometimes the bundle order causes very slow bundle resolution, see: https://community.openhab.org/t/starting-demo-app-takes-ages-10-12min-on-every-run-osgi-startup/122083

File: launch/app/app.bndrun (hunk 1 above)
Comment (holgerfriedrich) [reply launch/app/app.bndrun:127]: I'll have a look. I was thought not to touch whatever the resolver creates ;-)

**Comment by holgerfriedrich** [comment]
@wborn diff looks much better now, hope I did not break anything....

**Review by wborn — APPROVED** [review]
Thank you!

**Pull request merged**